src/main/java/com/bloxbean/cardano/client/coinselection/UtxoSelectionStrategy.java

```diff
@@ -12,42 +15,27 @@
  */
 public interface UtxoSelectionStrategy {
 
-    /**
-     * Selected utxos based on a strategy
-     * @param address Address to select utxos for
-     * @param unit Unit
-     * @param amount Amount
-     * @param excludeUtxos Utxos to ignore
-     * @return List of Utxos
-     */
-    List<Utxo> selectUtxos(String address, String unit, BigInteger amount, Set<Utxo> excludeUtxos) throws ApiException;
+    int DEFAULT_COIN_SELECTION_LIMIT = 20;
```

Comment:
We can keep DEFAULT_COIN_SELECTION_LIMIT constant in Configuration class with a setter in case application wants to set to higher or lower value. What do you think ?

Similarly, the default selection strategy can be kept in Configuration. So the default strategy can be changed through Configuration by the application. Currently, the default strategy is created inside UtxoTransactionBuilderImpl's constructor.

Reply:
Have moved `DEFAULT_COIN_SELECTION_LIMIT` to `Configuration` but `DefaultUtxoSelectionStrategyImpl` requires a utxoSevice to construct.